hypothesis_bio/pdb.py

```diff
@@ -0,0 +1,286 @@
+# -*- coding: utf-8 -*-
+
+"""Main module."""
+
+from datetime import date
+
+from hypothesis.strategies import (
+    booleans,
+    characters,
+    composite,
+    dates,
+    floats,
+    integers,
+    sampled_from,
+    text,
+)
+
+ACHAR = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
Take a look as `string.ascii_letters` in the stdlib for this

Reply:
Cool. Switched to using the stdlib thing.